endpoints/openrtb2/auction.go

```diff
@@ -1334,6 +1339,36 @@ func validateDevice(device *openrtb2.Device) error {
 	return nil
 }
 
+func validateOrFillChannel(requestPrebid *openrtb_ext.ExtRequestPrebid, requestExt *openrtb_ext.RequestExt, reqWrapper *openrtb_ext.RequestWrapper, isAmp bool) error {
```

Comment:
Do we need to pass in `requestPrebid` and `requestExt` to this function? We can just access them via the request wrapper's `GetRequestExt()` and `GetPrebid()` methods in this function. This will reduce the number of parameters in this function

Reply:
This will be updated to reduce the number of params!